feature-detects/svg/inline.js

```diff
@@ -7,8 +7,12 @@
   "notes": [{
     "name": "Test page",
     "href": "http://paulirish.com/demo/inline-svg"
+  }, {
+    "name: "Test page and results",
```

Comment:
you have a typo here that is breaking the parsing and thus the tests

`"name:` should be `"name":`